docs/setup_guides/hpc.rst

```diff
@@ -38,27 +41,27 @@ Create a new conda environment for our pangeo work:
 ::
 
     conda create -n pangeo -c conda-forge \
-        python=3.6 dask distributed xarray jupyterlab mpi4py
+        python=3.6 dask distributed xarray jupyterlab mpi4py dask_jobqueue
```

Comment:
This will be possible soon: https://github.com/conda-forge/staged-recipes/pull/5844